qt/applications/workbench/workbench/plotting/figurewindow.py

```diff
@@ -83,6 +86,12 @@ def dropEvent(self, event):
         self._plot_on_here(event.mimeData().text().split('\n'))
         QMainWindow.dropEvent(self, event)
 
+    def eventFilter(self, obj, event):
```

Comment:
For some reason the context menu is now not getting emitted.

Is there any reason we can't use the existing `def event(self, event)` above and just additionally check for `type == QEvent.ContextMenu` in there rather than needing an event filter? This seems to work based on a local check I have done.